src/test/kotlin/uk/gov/dluhc/printapi/messaging/SendApplicationToPrintMessageListenerIntegrationTest.kt

```diff
@@ -115,49 +117,35 @@ internal class SendApplicationToPrintMessageListenerIntegrationTest : Integratio
             val response = certificateRepository.findAll()
             assertThat(response).hasSize(1)
             val saved = response[0]
+            val instantEqualToRoundedSeconds: BiPredicate<Instant, Instant> =
```

Comment:
Value returned from the database is only to second precision - not the millis that is in the expected object.